tests/config.ini

```diff
@@ -11,6 +11,7 @@ verbose-http-errors = true
 agent-name = "DUNE Test Node"
 net-threads = 2
 max-transaction-time = 100
+read-only-read-window-time-us = 120000
```

Comment:
What is the purpose of this config? Seems like some odd configurations. Why `eos-vm` instead of `eos-vm-jit`? Why such a small `abi-serializer-max-time-ms`. Why specify `net-threads = 2` ? Why `max-transaction-time` of `100`? Why not shutdown when disk is full?

Reply:
@larryk85 can you help here?
You have added scripts/config.ini (and tests/config.ini is basically a copy of it used for tests) 1 year ago, so I'm not sure if you remember.

scripts/config.ini is used as default config when starting up nodeos in DUNE container.
tests/config.ini is used when starting up some examples nodes for integration tests.

Reply:
If memory serves correctly, most of the contents of this config aren't especially relevant. What is relevant here is that the address in this config match those in `tests/test_nodes.py` but are different from `scripts/config.ini`.

There is some calue in capturing that info into this config file or a readme in the test scripts directory.